app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CSVUploadForm
+  include ActiveModel::Validations
+
+  attr_reader :file
+
+  validates_with CSVValidator
+
+  def initialize(user)
+    @user = user
```

Comment:
```suggestion
  def initialize(current_user)
    @current_user = current_user
```

I recommend sticking with `current_user` to add more context (the cases when we have different users than the current users are quite common too so this helps be sure about which user it is :) )